contracts/NormalizedApi3ReaderProxyV1.sol

```diff
@@ -2,41 +2,57 @@
 pragma solidity ^0.8.27;
 
 import "./interfaces/INormalizedApi3ReaderProxyV1.sol";
-import "./ProxyUtils.sol";
 
 /// @title An immutable proxy contract that converts a Chainlink
 /// AggregatorV2V3Interface feed output to 18 decimals to conform with
-/// IApi3ReaderProxy decimal standard
+/// IApi3ReaderProxy decimal convention
 /// @dev This contract implements the AggregatorV2V3Interface to be compatible
 /// with Chainlink aggregators. This allows the contract to be used as a drop-in
 /// replacement for Chainlink aggregators in existing dApps.
 /// Refer to https://github.com/api3dao/migrate-from-chainlink-to-api3 for more
 /// information about the Chainlink interface implementation.
 contract NormalizedApi3ReaderProxyV1 is INormalizedApi3ReaderProxyV1 {
-    using ProxyUtils for int256;
-
     /// @notice Chainlink AggregatorV2V3Interface contract address
     address public immutable override feed;
 
-    uint8 internal immutable feedDecimals;
+    /// @dev Pre-calculated factor for scaling to 18 decimals.
+    int256 private immutable scalingFactor;
```

Comment:
Why not public?

Reply:
They could be (both `scalingFactor` and `isUpscaling`) but underlying feed exposes `decimals()` (and so does this one even though 18 is implicit)

Reply:
Yeah, I'd expose them (because I don't see a reason why not).